event_dictionary/0.6.0-DRAFT/examples/business/printFulfilment.example.json

```diff
@@ -0,0 +1,22 @@
+{
+  "header": {
+    "version": "0.6.0-DRAFT",
+    "topic": "event_print-fulfilment",
+    "source": "non pariatur elit officia sed",
+    "channel": "RM",
+    "dateTime": "1970-01-01T00:00:00.000Z",
+    "messageId": "3883af91-0052-4497-9805-3238544fcf8a",
+    "correlationId": "3883af91-0052-4497-9805-3238544fcf8a",
+    "originatingUser": "foo.bar@ons.gov.uk"
+  },
+  "payload": {
+    "printFulfilment": {
+      "caseId": "3883af91-0052-4497-9805-3238544fcf8a",
+      "packCode": "ut",
+      "personalisation": {
+        "firstName": "Joe",
+        "lastName": "Bloggs"
+      }
+    }
+  }
+}
```

Comment:
This file is where the actual change over the current `0.5.0` schema is, the added `personalisation` object.